.github/workflows/generate-used-icons.yml

```diff
@@ -0,0 +1,54 @@
+name: 🤖Generate used icons
+on:
+  workflow_dispatch:
+    dryRun:
+      type: boolean
+      required: false
+      description: --dry-run
+
+jobs:
+  generate-icons-array:
```

Comment:
you can push directly to your branch if there are any changes

`.github/workflows/auto-update-icons.yml`

Reply:
I moved the command run to `auto-update-icons`  (I hope I got you right)